clients/include/rocblas_arguments.hpp

```diff
@@ -62,9 +62,9 @@ struct Arguments
     rocblas_int stride_b; //  stride_b > transB == 'N' ? ldb * N : ldb * K
     rocblas_int stride_c; //  stride_c > ldc * N
     rocblas_int stride_d; //  stride_d > ldd * N
-
     rocblas_int stride_x;
     rocblas_int stride_y;
+    rocblas_int stride_scale;
```

Comment:
We don't want to add this variable to the Arguments struct. It is just a intermediate variable only found in the yaml files.

Reply:
> 
> 
> We don't want to add this variable to the Arguments struct. It is just a intermediate variable only found in the yaml files.

Correct, `Arguments` is already pretty bloated, and adding members to `Arguments` requires adding them to `rocblas_common.yaml` as well.

Reply:
okay I was logging it so I could understand the tests better.   I can remove it and rename it to strideScale as in other PR, but if we don't log it should have a different convention ?

Reply:
We could reverse-compute it from the other arguments and log it, assuming it's deterministic.